synapseclient/models/protocols/json_schema_protocol.py

```diff
@@ -490,99 +553,26 @@ def get_schema_validation_statistics(
 
             ```python
-                import synapseclient
-                from synapseclient.models import File, Folder
-
-                syn = synapseclient.Synapse()
-                syn.login()
-
-                # Define Project and JSON schema info
-                PROJECT_ID = syn.findEntityId(name="test_json_schema_project")  # Replace with your project name
-                ORG_NAME = "UniqueOrg"  # Replace with your organization name
-                SCHEMA_NAME = "myTestSchema"  # Replace with your schema name
-                VERSION = "0.0.1"
-                SCHEMA_URI = f"{ORG_NAME}-{SCHEMA_NAME}-{VERSION}"
-
-                # Create organization (if not already created)
-                js = syn.service("json_schema")
-                all_orgs = js.list_organizations()
-                for org in all_orgs:
-                    if org["name"] == ORG_NAME:
-                        print(f"Organization {ORG_NAME} already exists.")
-                        break
-                else:
-                    print(f"Creating organization {ORG_NAME}.")
-                    js.create_organization(ORG_NAME)
-
-                # Create the schema (if not already created)
-                schema_definition = {
-                    "$id": "mySchema",
-                    "type": "object",
-                    "properties": {
-                        "foo": {"type": "string"},
-                        "baz": {"type": "string", "const": "example_value"},  # Example constant for derived annotation
-                        "bar": {"type": "integer"},
-                    },
-                    "required": ["foo"]
-                }
-
-                my_test_org = js.JsonSchemaOrganization(ORG_NAME)
-                test_schema = my_test_org.get_json_schema(SCHEMA_NAME)
-                if not test_schema:
-                    test_schema = my_test_org.create_json_schema(schema_definition, SCHEMA_NAME, VERSION)
-
-                # Create a test folder
-                test_folder = Folder(name="test_script_folder", parent_id=PROJECT_ID)
-                test_folder.store()
-
-                # Bind JSON schema to the folder
-                bound_schema = test_folder.bind_schema(
-                    json_schema_uri=SCHEMA_URI,
-                    enable_derived_annotations=True
-                )
-
-                # Validate the folder entity against the bound schema
-                test_folder.annotations = {"foo": "test_value_new", "bar": 42}  # Example annotations
-                test_folder.store()
-
-                # Get schema validation statistics
-                stats = test_folder.get_schema_validation_statistics()
-                print('Validation statistics:', stats)
-            ```
-        """
-        return JSONSchemaValidationStatistics()
-
-    def get_invalid_validation(
-        self, *, synapse_client: Optional["Synapse"] = None
-    ) -> Generator["InvalidJSONSchemaValidation", None, None]:
```

Comment:
This function was removed from `synapseclient/models/protocols/json_schema_protocol.py` because it already exists in `synapseclient/models/mixins/json_schema.py`